test/builder/pipeline_test.go

```diff
@@ -124,6 +124,54 @@ func TestPipelineRun(t *testing.T) {
 	}
 }
 
+func TestPipelineRunWithRetry(t *testing.T) {
+	startTime := time.Now()
+	pipelineRun := tb.PipelineRun("pear", "foo", tb.PipelineRunSpec(
+		"tomatoes", tb.PipelineRunServiceAccount("sa"),
+		tb.PipelineRunParam("first-param", "first-value"),
+		tb.PipelineRunTimeout(&metav1.Duration{Duration: 1 * time.Hour}),
+		tb.PipelineRunRetries(1),
```

Comment:
what do you think about updating the test case above to include a retry instead of adding a completely new test case? since these tests are just about the structure of the resulting pipelinerun, i think that might be simpler (and it would be untenable to test all the combinations of permutations!)